scripts/setup/apt-repos/zulip/custom.sh

```diff
@@ -26,10 +27,15 @@ if [[ ! -e /usr/share/doc/groonga-apt-source/copyright ]]; then
             read -r distribution
             read -r release
         } <<<"$os_info"
+
         if [ "$distribution" = debian ] && [ "$release" = bookworm ]; then
-            # PGroonga binaries are not yet provided for Debian 12.
-            exit
+            arrow_apt_source_deb="apache-arrow-apt-source-latest-$release.deb"
```

Comment:
Can you add a comment on why we need this dependency?

Reply:
Added.